pages/storage-page.ts

```diff
@@ -0,0 +1,16 @@
+// Select LVM as the storage device
```

Comment:
no comment needed, it is self-explanatory that this map the storage page, otherwise we would need to revisit this in the future